QuickshaReTests/QRPictoreGeneratorTests.swift

```diff
@@ -19,7 +19,7 @@ class QRPictoreGeneratorTests: XCTestCase {
         let picture = generator.qrPicture(for: message)
         
         let detector = CIDetector(ofType: CIDetectorTypeQRCode, context: nil)!
-        let features = detector.features(in: CIImage(image: picture.uiImage)!) as! [CIQRCodeFeature] //swiftlint:disable:this force_cast
+        let features = detector.features(in: CIImage(image: picture.uiImage)!) as! [CIQRCodeFeature] // swiftlint:disable:this force_cast
```

Comment:
<!--
  0 failure: 
  1 warning:  Line should be 12...
  
  
  DangerID: danger-id-Danger;
  File: QuickshaReTests/QRPictoreGeneratorTests.swift;
  Line: 22;
-->

- :warning: Line should be 120 characters or less: currently 137 characters (`line_length`)